code/user-clients/nexus/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "nexus",
+  "version": "1.0.0",
+  "description": "FFD Request Nexus",
+  "main": "server.js",
+  "scripts": {
+    "test": "npm test",
+    "start": "nodemon server.js"
```

Comment:
Here's a dank pro tip: https://github.com/DanielSeehausen/fi-field-day/blob/master/code/node-server/package.json#L9

You can add hot reloading to ur server AND debug it using the chrome devtools with the `--inspect` flag

https://medium.com/the-node-js-collection/debugging-node-js-with-google-chrome-4965b5f910f4


![bitmoji](https://render.bitstrips.com/v2/cpanel/e44376a0-4919-4b1c-8207-e8c2953ac144-7c243825-53cb-4c20-a7d9-3fc8d332e86d-v1.png?transparent=1&palette=1&width=246)